layers/src/layer-publisher-stack.ts

```diff
@@ -36,6 +36,8 @@ export class LayerPublisherStack extends Stack {
         Runtime.NODEJS_16_X,
         Runtime.NODEJS_18_X
       ],
+      license: 'MIT-0',
+      compatibleArchitectures: [Architecture.X86_64],
```

Comment:
What do you think what make us comfortable to add ARM here?

My understanding is that with the current dependency set and with the fact that this is Node.js, our packages should be supported for ARM without requiring any change.

I do however think that it'd be a good idea to test this. Do you think this should be something that is tested in the Layer integration tests, or in the beta deployment, or in the canary (that we want to add), or elsewhere?

Reply:
Adding ARM support would require a dedicated canary, in addition what we have planned for the layer publishing workflow #1355 . I'd address ARM support in a separate issue and PR. 

Do you want me to extend tests for license and architecture?